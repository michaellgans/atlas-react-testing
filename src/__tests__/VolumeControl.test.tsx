// Test for VolumeControl Component

// Imports
import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import { VolumeControl } from "@/components/VolumeControl";
import { ContextProvider } from "@/components/ContextProvider";

test("VolumeControl renders correctly", () => {
    const {asFragment} = render(
        <ContextProvider>
            <VolumeControl />
        </ContextProvider>
    );
    expect(asFragment).toMatchSnapshot();
});
