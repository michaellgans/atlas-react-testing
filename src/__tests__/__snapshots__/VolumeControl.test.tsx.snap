// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`VolumeControl renders correctly 1`] = `[Function]`;
